? c/src/lib/libcpu/powerpc/mpc6xx/mmu/mmu_support.c
? c/src/lib/libcpu/powerpc/mpc6xx/mmu/mmu_support.h
? cpukit/libmmu
? testsuites/libtests/mmutests
Index: c/src/lib/libbsp/powerpc/psim/startup/bspstart.c
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libbsp/powerpc/psim/startup/bspstart.c,v
retrieving revision 1.36
diff -u -U N -r1.36 bspstart.c
Index: c/src/lib/libbsp/powerpc/psim/startup/device-tree
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libbsp/powerpc/psim/startup/device-tree,v
retrieving revision 1.2
diff -u -U N -r1.2 device-tree
Index: c/src/lib/libbsp/powerpc/psim/startup/linkcmds
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libbsp/powerpc/psim/startup/linkcmds,v
retrieving revision 1.31
diff -u -U N -r1.31 linkcmds
Index: c/src/lib/libcpu/powerpc/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libcpu/powerpc/Makefile.am,v
retrieving revision 1.54
diff -u -U N -r1.54 Makefile.am
Index: c/src/lib/libcpu/powerpc/preinstall.am
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libcpu/powerpc/preinstall.am,v
retrieving revision 1.23
diff -u -U N -r1.23 preinstall.am
Index: c/src/lib/libcpu/powerpc/new-exceptions/bspsupport/vectors_init.c
===================================================================
RCS file: /usr1/CVS/rtems/c/src/lib/libcpu/powerpc/new-exceptions/bspsupport/vectors_init.c,v
retrieving revision 1.9
diff -u -U N -r1.9 vectors_init.c
Index: cpukit/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/Makefile.am,v
retrieving revision 1.86
diff -u -U N -r1.86 Makefile.am
Index: cpukit/configure.ac
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/configure.ac,v
retrieving revision 1.162
diff -u -U N -r1.162 configure.ac
Index: cpukit/preinstall.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/preinstall.am,v
retrieving revision 1.37
diff -u -U N -r1.37 preinstall.am
Index: cpukit/wrapup/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/cpukit/wrapup/Makefile.am,v
retrieving revision 1.66
diff -u -U N -r1.66 Makefile.am
Index: make/custom/psim.cfg
===================================================================
RCS file: /usr1/CVS/rtems/make/custom/psim.cfg,v
retrieving revision 1.32
diff -u -U N -r1.32 psim.cfg
Index: testsuites/libtests/Makefile.am
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/libtests/Makefile.am,v
retrieving revision 1.14
diff -u -U N -r1.14 Makefile.am
Index: testsuites/libtests/configure.ac
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/libtests/configure.ac,v
retrieving revision 1.21
diff -u -U N -r1.21 configure.ac
Index: testsuites/samples/hello/init.c
===================================================================
RCS file: /usr1/CVS/rtems/testsuites/samples/hello/init.c,v
retrieving revision 1.15
diff -u -U N -r1.15 init.c
